src/code42cli/cmds/search/cursor_store.py

```diff
@@ -75,13 +76,31 @@ def __init__(self, profile_name):
         super().__init__(dir_path)
 
 
-def get_file_event_cursor_store(profile_name):
-    return FileEventCursorStore(profile_name)
+class AuditLogCursorStore(BaseCursorStore):
+    def __init__(self, profile_name):
+        dir_path = get_user_project_path("audit_log_checkpoints", profile_name)
+        super().__init__(dir_path)
 
+    def get_events(self, cursor_name):
+        try:
+            location = path.join(self._dir_path, cursor_name) + "_events"
+            with open(location) as checkpoint:
+                try:
+                    return json.loads(checkpoint.read())
+                except json.JSONDecodeError:
+                    return []
+        except FileNotFoundError:
+            return []
 
-def get_alert_cursor_store(profile_name):
-    return AlertCursorStore(profile_name)
+    def replace_events(self, cursor_name, new_events):
+        location = path.join(self._dir_path, cursor_name) + "_events"
+        with open(location, "w") as checkpoint:
+            return checkpoint.write(json.dumps(new_events))
```

Comment:
It does not seem like this return value gets used anywhere

Reply:
it was like that on the parent class `replace()` method too, just copy/pasted. I've removed return from both.